lib/flipper/adapters/memcached.rb

```diff
@@ -0,0 +1,74 @@
+require 'dalli'
+
+module Flipper
+  module Adapters
+    class Memcached
+      include Flipper::Adapter
+      FeaturesKey =  :flipper_features
+
+      attr_accessor :cache
+      attr_accessor :name
+
+      def initialize(adapter, address = 'localhost:11211', options = {})
```

Comment:
I'm kind of averse to taking parameters and then initializing an instance for people. The main problem I've ran into in the past is that these initialization options change and then I'm constantly updating or behind what is possible. Additionally, people usually have an instance already they are using in their app, so this forces multiple connections to be created. Instead I usually do something like this:

```ruby
def initialize(adapter, cache)
  @adapter = adapter
  @cache = cache
end
```